bot/utils/converters.py

```diff
@@ -0,0 +1,16 @@
+import discord
+from discord.ext.commands.converter import MessageConverter
+
+
+class BetterMessageConverter(MessageConverter):
```

Comment:
Perhaps a more descriptive name such as `MessageLinkConverter`?

Reply:
Perhaps, although "MessageLinkConverter" doesn't really match IMO, since this also handles message ID's, and pairs of ChannelID-MessageID. I couldn't think of a better name. WrappedMessageConverter?

Reply:
Hmm that's true. `WrappedMessageConverter` is probably fine then.